server/schedule/operator.go

```diff
@@ -45,6 +46,17 @@ func (tl TransferLeader) IsFinish(region *core.RegionInfo) bool {
 	return region.Leader.GetStoreId() == tl.ToStore
 }
 
+// Influence calculates the store difference that current step make
+func (tl TransferLeader) Influence(opInfluence OpInfluence, region *core.RegionInfo) {
+	from := opInfluence.GetStoreInfluence(tl.FromStore)
+	to := opInfluence.GetStoreInfluence(tl.ToStore)
+
+	from.LeaderSize -= int(region.ApproximateSize)
```

Comment:
need to check < 0 here?

Reply:
check ApproximateSize < 0 ?

Reply:
need to check leadersize < approximate size ?

Reply:
opInfluence is difference value, negative here means operator will make leaderSize decrease.

Reply:
got it